app/gui2/src/components/visualizations/TableVisualization.vue

```diff
@@ -407,7 +407,7 @@ function toLinkField(fieldName: string): ColDef {
     tooltipValueGetter: () => {
       return `Double click to view this ${newNodeSelectorValues.value.tooltipValue} in a separate component`
     },
-    cellRenderer: (params: any) => `<a href='#'> ${params.value} </a>`,
+    cellRenderer: (params: any) => `<a> ${params.value} </a>`,
```

Comment:
it's just providing the link styling to the text in the cell

Reply:
it was brought up that tag selectors (`a` instead of `.link` for example) apparently are less performant - maybe this could be changed to a `<span class="link">`?